### ISSUE TITLE
export of property sets / properties needs to reference the element

### ISSUE THREAD

**Issue by MarkusSteinbrecher** [bldrs-ai/ifctool#41]
Did some testing with project [Seestrasse](https://drive.google.com/file/d/14T04bi0SAaOXAB05kAMr2aoB4gdk8cHv/view?usp=sharing) and exported to csv using: npx @bldrs-ai/ifctool@latest Seestrassev2.ifc --out=csv >test1.csv 

The results are [here](https://docs.google.com/spreadsheets/d/1sH-U91mZWMIuejLyDHfWeRVd-YCrhAsPzNebgj8xxQo/edit#gid=112694239).

The properties are exported, e.g. length of a wall but I do not understand the way they are structured. The entries do not seem to be linked to the element they belong to, or am I missing it?

**Comment by pablo-mayrgundter** [comment]
Sorta have it working locally.. but yeah, it's that the pset object is deeply nested so you get kind of a useless table.  will see what i can do

<img width="969" alt="image" src="https://user-images.githubusercontent.com/2480879/198359975-7ba21220-2df9-412b-8d71-d151798e5df7.png">

**Comment by MarkusSteinbrecher** [comment]
I did a another csv export of the latest Seestrasse model in Blender across element types (Space, Wall, Window, Door, Slab) and with a couple of the important properties: [Seestrasse Dashboard Export](https://docs.google.com/spreadsheets/d/1JzH7BYjaicEdHQttVKDVKr2koabrvfBQaZxY1NIa2HA/edit#gid=1929868433) ... Table will get sparser with more properties but totally doable, if a set of properties and element types is selected. PS: this does take about 2 Minutes and spikes the blender process :)

**Comment by pablo-mayrgundter** [comment]
Thanks..  did you have to select element types to export?  I did see an example in ifcjs that does explicitly list common types for export.  I wonder if that's the simplest way

**Comment by MarkusSteinbrecher** [comment]
yes, you have to list the types and properties in Blender to export
